.github/workflows/trivy-vuln-scanner.yaml

```diff
@@ -1,4 +1,4 @@
-name: Vulnerability Scanner
+name: Vulnerability Scan
```

Comment:
Waar komt deze file van daan? Hij staat al in main want dit is alleen een wijziging. Toch kan ik het niet zien in de code...

Reply:
Ja dat klopt, dit was een foutje. Ik zag dat ik nog aan het werken was op de main branch en had de code gepushed naar main. Dit heb ik vervolgens weer verwijderd en een nieuwe branch aangemaakt waar ik de code in heb gepushed. Hier maakte hij een PR van. Ik snap alleen nog niet waarom ik direct op de main branch iets kan pushen en er geen PR wordt aangemaakt, aangezien er een lock op zit?

Reply:
Dit zou een optie kunnen zijn
![afbeelding](https://github.com/user-attachments/assets/faa7a121-2a7e-4675-a43c-cf9b3975b953)

Reply:
Ja, we moeten dit echt dicht zetten. Door op de main branch te pushen ga je om de work-flow heen voor reviewen van PRs. Doen dus 👍